package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "text-editor",
+  "widgetName": "TextEditor",
+  "version": "0.1.0",
+  "description": "Rich inline text editing",
+  "scripts": {
+    "pretest": "tsc",
+    "test": "karma start --single-run --codeCoverage && node tests/remap.js",
+    "lint": "tslint -c tslint.json '**/*.ts' '**/*.tsx' --exclude '**/node_modules/**'"
+  },
+  "pre-commit": [
+    "pretest",
+    "lint"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendixlabs/TextEditor"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/mendixlabs/TextEditor/issues"
+  },
+  "devDependencies": {
+    "@types/classnames": "^2.2.0",
+    "@types/core-js": "0.9.36",
+    "@types/draft-js": "^0.10.1",
+    "@types/enzyme": "^2.8.0",
+    "@types/jasmine": "^2.5.51",
+    "@types/karma": "^0.13.35",
+    "@types/react": "^15.0.27",
+    "@types/react-dom": "^15.5.0",
+    "check-dependencies": "^1.0.1",
+    "copy-webpack-plugin": "^4.0.1",
+    "core-js": "^2.4.1",
```

Comment:
Couldn't get the widget to work in I.E using core-js. The core-js polyfils are not loaded early enough, so they're always missing in I.E. Perhaps u know how to make it work.